diff --git a/native_modules.gradle b/native_modules.gradle
index 58492ea5d5410cd2cb362385a8ebb8754d27c301..d03521c86f9a3638fa4646088ef3aabde7eb5b6d 100644
--- a/native_modules.gradle
+++ b/native_modules.gradle
@@ -298,6 +298,7 @@ class ReactNativeModules {
         if (it.cxxModuleCMakeListsModuleName != null) {
           autolinkedLibraries += "\n${it.cxxModuleCMakeListsModuleName}"
         }
+        autolinkedLibraries
       }.minus(null).join('\n  ')
     }
 
