diff --git a/node_modules/react-native/React/Base/RCTBridge+Inspector.h b/node_modules/react-native/React/Base/RCTBridge+Inspector.h
new file mode 100644
index 0000000..54d8db3
--- /dev/null
+++ b/node_modules/react-native/React/Base/RCTBridge+Inspector.h
@@ -0,0 +1,30 @@
+/*
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+#import <React/RCTBridge.h>
+
+#ifdef __cplusplus
+#import <jsinspector-modern/ReactCdp.h>
+#endif
+
+@interface RCTBridge (Inspector)
+
+/**
+ * The HostTarget for this bridge, if one has been created. Exposed for RCTCxxBridge only.
+ */
+@property (nonatomic, assign, readonly)
+#ifdef __cplusplus
+    facebook::react::jsinspector_modern::PageTarget *
+#else
+    // The inspector infrastructure cannot be used in C or Swift code.
+    void *
+#endif
+        inspectorTarget;
+
+@property (nonatomic, readonly, getter=isInspectable) BOOL inspectable;
+
+@end
\ No newline at end of file
diff --git a/node_modules/react-native/React/Base/RCTBridge+Private.h b/node_modules/react-native/React/Base/RCTBridge+Private.h
index d9b2be4..4d03df3 100644
--- a/node_modules/react-native/React/Base/RCTBridge+Private.h
+++ b/node_modules/react-native/React/Base/RCTBridge+Private.h
@@ -6,9 +6,6 @@
  */
 
 #import <React/RCTBridge.h>
-#ifdef __cplusplus
-#import <jsinspector-modern/ReactCdp.h>
-#endif
 
 @class RCTModuleRegistry;
 @class RCTModuleData;
@@ -73,17 +70,6 @@ RCT_EXTERN void RCTRegisterModule(Class);
  */
 @property (nonatomic, strong, readonly) RCTModuleRegistry *moduleRegistry;
 
-/**
- * The page target for this bridge, if one has been created. Exposed for RCTCxxBridge only.
- */
-@property (nonatomic, assign, readonly)
-#ifdef __cplusplus
-    facebook::react::jsinspector_modern::PageTarget *
-#else
-    // The inspector infrastructure cannot be used in C code.
-    void *
-#endif
-        inspectorTarget;
 @end
 
 @interface RCTBridge (RCTCxxBridge)
@@ -155,12 +141,6 @@ RCT_EXTERN void RCTRegisterModule(Class);
 
 @end
 
-@interface RCTBridge (Inspector)
-
-@property (nonatomic, readonly, getter=isInspectable) BOOL inspectable;
-
-@end
-
 @interface RCTCxxBridge : RCTBridge
 
 // TODO(cjhopman): this seems unsafe unless we require that it is only called on the main js queue.
diff --git a/node_modules/react-native/React/Base/RCTBridge.mm b/node_modules/react-native/React/Base/RCTBridge.mm
index 65d827e..9c18905 100644
--- a/node_modules/react-native/React/Base/RCTBridge.mm
+++ b/node_modules/react-native/React/Base/RCTBridge.mm
@@ -6,6 +6,7 @@
  */
 
 #import "RCTBridge.h"
+#import "RCTBridge+Inspector.h"
 #import "RCTBridge+Private.h"
 
 #import <objc/runtime.h>
diff --git a/node_modules/react-native/React/CoreModules/RCTDevSettings.mm b/node_modules/react-native/React/CoreModules/RCTDevSettings.mm
index b5e81c9..1093050 100644
--- a/node_modules/react-native/React/CoreModules/RCTDevSettings.mm
+++ b/node_modules/react-native/React/CoreModules/RCTDevSettings.mm
@@ -10,6 +10,7 @@
 #import <objc/runtime.h>
 
 #import <FBReactNativeSpec/FBReactNativeSpec.h>
+#import <React/RCTBridge+Inspector.h>
 #import <React/RCTBridge+Private.h>
 #import <React/RCTBridgeModule.h>
 #import <React/RCTConstants.h>
diff --git a/node_modules/react-native/React/CxxBridge/RCTCxxBridge.mm b/node_modules/react-native/React/CxxBridge/RCTCxxBridge.mm
index a611708..05278d5 100644
--- a/node_modules/react-native/React/CxxBridge/RCTCxxBridge.mm
+++ b/node_modules/react-native/React/CxxBridge/RCTCxxBridge.mm
@@ -9,6 +9,7 @@
 #include <future>
 
 #import <React/RCTAssert.h>
+#import <React/RCTBridge+Inspector.h>
 #import <React/RCTBridge+Private.h>
 #import <React/RCTBridge.h>
 #import <React/RCTBridgeMethod.h>
diff --git a/node_modules/react-native/ReactCommon/react/runtime/platform/ios/ReactCommon/RCTInstance.mm b/node_modules/react-native/ReactCommon/react/runtime/platform/ios/ReactCommon/RCTInstance.mm
index a200998..918bd48 100644
--- a/node_modules/react-native/ReactCommon/react/runtime/platform/ios/ReactCommon/RCTInstance.mm
+++ b/node_modules/react-native/ReactCommon/react/runtime/platform/ios/ReactCommon/RCTInstance.mm
@@ -11,6 +11,7 @@
 
 #import <React/NSDataBigString.h>
 #import <React/RCTAssert.h>
+#import <React/RCTBridge+Inspector.h>
 #import <React/RCTBridge+Private.h>
 #import <React/RCTBridge.h>
 #import <React/RCTBridgeModule.h>
